src/main/java/com/comphenix/protocol/injector/StructureCache.java

```diff
@@ -127,7 +127,7 @@ public static StructureModifier<Object> getStructure(final PacketType packetType
 	 *
 	 * @return an accessor to a constructor which creates a data serializer.
 	 */
-	private static ConstructorAccessor getTrickDataSerializerOrNull() {
+	public static ConstructorAccessor getTrickDataSerializerOrNull() {
```

Comment:
Might be better to expose a method which creates a serializer instance, rather than exposing the constructor?